src/main/java/carRacing/Domain/Vehicles.java

```diff
@@ -0,0 +1,45 @@
+package carRacing.Domain;
+
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Vehicles {
+
+    List<Vehicle> vehicles;
+
+    private Vehicles(List<Vehicle> vehicles) {
+        this.vehicles = vehicles;
+    }
+
+    public static Vehicles create(List<Vehicle> vehicles) {
+        return new Vehicles(vehicles);
+    }
+
+    public Vehicles move() {
+        for (Vehicle vehicle : vehicles) {
+            vehicle.move();
+        }
+        return create(vehicles);
+    }
+
+    private int winnerPosition() {
+        int max = 0;
+        for (Vehicle vehicle : vehicles) {
+            max = vehicle.greaterThen(max);
+        }
+        return max;
+    }
+
+    public List<String> findWinner() {
+        int max = winnerPosition();
+        return vehicles.stream().filter(v -> {
+            return v.inquiryPosition() >= max;
+        }).map(Vehicle::getName).collect(Collectors.toList());
```

Comment:
가독성 측면에서 한 라인에 하나의 스트림 람다식이 배치될 수 있도록 하면 좋을 것 같네요 :)
```suggestion
                return vehicles.stream()
                .filter(v -> v.inquiryPosition() >= max)
                .map(Vehicle::getName)
                .collect(Collectors.toList());
```